singleFrameProcCoreTasks/isrtask.rst

```diff
@@ -0,0 +1,432 @@
+
+#######
+IsrTask 
+#######
+
+Instrument Signature Removal (ISR) is a sequence of steps taken to
+'clean' images of various aspects of defects that any system of optics
+and detectors will imprint on an image by default, and is generally
+one of the very first procedures carried out on an exposure.
```

Comment:
Ok... how does these changes sound:

"Instrument Signature Removal (ISR) is a sequence of steps taken to
correct the effects imprinted on the counts coming out of the readout
by the physical characteristics of the detector and the electronics of
the readout chain.  It is generally the very first procedure carried
out on an exposure."